boards/stm32f3discovery/Makefile

```diff
@@ -22,6 +18,4 @@ flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 
 .PHONY: program
 program: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
-	arm-none-eabi-objcopy --update-section .apps=$(APP) $(KERNEL) $(KERNEL_WITH_APP)
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "init; reset halt; flash write_image erase $(KERNEL_WITH_APP); verify_image $(KERNEL_WITH_APP); reset; shutdown"
-	
```

Comment:
This should be in a separate PR

Reply:
It was for testing only and I deleted it when the PR was done

Reply:
It's still in this PR though, it makes it very hard to review when you add code then remove it later in the same PR. It also makes it harder to bisect regressions once this is merged. Can you please try to avoid doing this and only keep the relevant code in the PR.

Reply:
I am so sorry @alistair23 . I cleaned up the branch and everything should be fine now.

Reply:
This is still in the PR. You have completely remove the ability to program the board.